pubspec.yaml

```diff
@@ -9,20 +9,19 @@ authors:
   - Eric Kok <erickok@gmail.com>
 
```

Comment:
Can you bump the version here to 1.2.0 as well? Then all we need for a release is to merge this PR.

Reply:
Done. Should be good to go!